index.js

```diff
@@ -18,24 +18,29 @@ function getSize(el) {
   }
 }
 
-function useComponentSize(ref) {
-  var _useState = useState(getSize(ref ? ref.current : {}))
+function useComponentSize(elemOrRef) {
+  const el = elemOrRef && (
```

Comment:
remove this statement altogether, and rename `elemOrRef` to `el` to make this a breaking change that drops support for `RefObject` inputs.

This would be better to make it clearer to users that passing in a `RefObject` is asking for trouble since it won't trigger a re-render